README.adoc

```diff
@@ -4690,6 +4690,31 @@ ary[..42]
 ary[0..42]
 ----
 
+=== Collection querying [[collection-querying]]
+
+When possible, use https://docs.ruby-lang.org/en/master/Enumerable.html#module-Enumerable-label-Methods+for+Querying[predicate methods from `Enumerable`] rather than expressions with `#count`.
+
+[source,ruby]
+----
+# bad
+array.count > 0
```

Comment:
Btw, there's also `length` to consider. ;-)

Reply:
Yep, updated. Note just that I wouldn't include `length`/`size` in the cop because it would yield too many false positives since Ruby has core classes which implement those methods, but aren't Enumerable (e.g. [Integer](https://rubyapi.org/3.4/o/integer#method-i-size), [String](https://rubyapi.org/3.4/o/string#method-i-size), [File](https://rubyapi.org/3.4/o/file#method-c-size), etc.).

Reply:
Yeah, I figured as much although probably this should be also mentioned in the cop's docs as well, and we can also provide "conservative/aggressive" modes there in case someone wants to do a broader search in their codebase (e.g. on a one off basis). Anyways, that's not relevant to the PR here.